diff --git a/ttt-api/src/index.ts b/ttt-api/src/index.ts
index 6549e7d..e10366f 100644
--- a/ttt-api/src/index.ts
+++ b/ttt-api/src/index.ts
@@ -1,26 +1,30 @@
+import express, { Response } from "express";
+import { Transaction } from "sequelize/types";
+
 import {
-  assertNever,
   coerce,
   coerceToGameState,
   coerceToMove,
   coerceToPlayer,
-  CoreGamePlaying,
-  CoreMove,
   CORE_GAME_OVER_TIE,
   CORE_GAME_OVER_WIN,
   CORE_GAME_PLAYING,
+  CoreGamePlaying,
+  CoreMove,
   createGame,
   findWin,
   isPosTaken,
   nextPlayer,
   resolveGame
 } from "@grancalavera/ttt-core";
-import express, { Response } from "express";
-import { Transaction } from "sequelize/types";
-import { ErrorCode, ErrorResponse, GameResponse, MovesResponse } from "./model";
-import { GameModel, MoveModel, create, toUnsafeMove } from "./store";
 
-const store = create("./store.sqlite");
+import { GameModel, MoveModel, store, toUnsafeMove } from "./store";
+import { GameResponse, ErrorCode, ErrorResponse, MovesResponse } from "./model";
+
+function assertNever(value: never): never {
+  throw new Error(`unexpected value ${value}`);
+}
+
 const app = express();
 app.use(express.json());
 
